<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hozzáad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Töröl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Módosít</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ár</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Típus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Keres</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>641</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Azonosító</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Márka</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Típus</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ár</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Alvázszám</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Eladás helye</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_price">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_brand">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Márka</string>
    </property>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Megnyit</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Mentés</string>
   </property>
  </action>
  <action name="actionMegnyit_s">
   <property name="text">
    <string>Megnyitás</string>
   </property>
  </action>
  <action name="actionMent_s">
   <property name="text">
    <string>Mentés</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
